ci_scripts/check_category_linking.rb

```diff
@@ -22,9 +22,17 @@
 all_headers = (Dir.glob("Stripe/PublicHeaders/*.h") + Dir.glob("Stripe/*.h")).map do |h|
   filename = File.basename(h)
 end
-categories = (all_headers.select do |h|
-  h.include?("+") && !h.include?("Private") && !h.include?("Fabric")
-end).sort
+
+def is_category(h)
+  if h.include?("+")
+    return !h.include?("Private") && !h.include?("Fabric")
```

Comment:
Yeah, the reliance on filenames in our current check is definitely brittle... updating to look at implementation files is a good call.

I think I'd still prefer not requiring adding `-ObjC` to use the static lib – Fabric uses the static lib, and I think adding an additional step to the "click to install" integration wouldn't be great.